lib/routes/content.js

```diff
@@ -358,10 +359,12 @@ const routes = application => {
         return next();
       }
 
-      // Validation
-      const validated = content.form.validate(req.body, type);
+      if (req.body.submit === 'Save Revision') {
```

Comment:
Can `Save Revision` be added to config so that we don't have to control what this text is in multiple places?